dlt/common/schema/utils.py

```diff
@@ -698,4 +698,4 @@ def standard_hints() -> Dict[TColumnHint, List[TSimpleRegex]]:
 
 
 def standard_type_detections() -> List[TTypeDetections]:
-    return ["timestamp", "iso_timestamp"]
+    return ["iso_timestamp"]
```

Comment:
Is this a concern / breaking change for any experienced users relying on dlt pipelines in production and its existing type detection?

Reply:
existing pipelines will keep the already inferred type and all will work the same. only when dataset or table is deleted and data is reloaded, the type will be different. do you rely in this anywhere? such timestamps happen most often in logs

Reply:
Ah okay cool, I was mostly wondering for the case of existing pipelines. I have some Mongodb/Timescaledb data pipelines which may inadvertently rely on this. Its nbd if this is no longer the default behavior but instead opt-in, just good to know! I can visualize many cases where this would not be desired but luckily, none of those ever happened to me :)